README.md

```diff
@@ -130,6 +133,10 @@ For example, this can be one commit for each of:
 
 Make sure that all checks still pass for every commit.
 
+When using dependabot to update dependencies (where there is nothing to group),
+you can trigger a new dependabot run (to potentially open additional pull requests)
+at [Insights > Dependency graph > Dependabot](https://github.com/wmde/new-lexeme-special-page/network/updates) > Recent update jobs > Check for updates.
```

Comment:
Nice, I wasn't aware that this is possible! However, from reading this paragraph, I'm not yet sure why I would want to do that. Can you elaborate?

Reply:
I guess it depends on the workflow you use. This time, I tried to do as many updates as seemed reasonable / justifiable using dependabot, to minimize the number of review round-trips (which take much longer than me just interacting with dependabot), especially since those can’t really be parallelized (I could upload several PRs, but merging one would probably invalidate the other and rebasing that would discard the review, so I’d still have to wait again, because GitHub is very inflexible about code review compared to Gerrit). But this kind of contradicts the process above where we want to “keep the number of commits … manageable” :/

Reply:
Ah makes sense, maybe we could add a sentence somewhere about "to make sure the existing PRs are fully up-to-date) or something similar?

Reply:
I rephrased it a bit.

Reply:
Better, thanks!